### PR TITLE
444-exporter-show-page

### PR DESCRIPTION
**Pull request by alishaevn**
ref #444 

# expected behavior

# demo
| | exporter show page | 
| --- | --- |
| works |image |
| collections | image |
| file sets | image|

### CODE REVIEW

**Review by ShanaLMoore — APPROVED** [review]

**Comment by ShanaLMoore** [comment]
The exporters show page would be easier to read if the tab sections were separated into partials. However I see that this is how the importers show page is done too.

**Comment by alishaevn** [comment]
> @alishaevn The exporters show page would be easier to read if the tab sections were separated into partials. However I see that this is how the importers show page is done too.

great idea! I've updated this. can you review again @ShanaLMoore?

File: app/views/bulkrax/exporters/show.html.erb [hunk 1]
```diff
@@ -87,49 +87,25 @@
...
-            <th>Identifier</th>
-            <th>Entry ID</th>
-            <th>Status</th>
-            <th>Errors</th>
-            <th>Status Set At</th>
-            <th>Actions</th>
-          </tr>
-        </thead>
-        <tbody>
-          <% @work_entries.each do |e| %>
```
Comment (alishaevn) [inline app/views/bulkrax/exporters/show.html.erb:109]: moved the entry tabs into partials.

File: app/views/bulkrax/importers/show.html.erb [hunk 2]
```diff
@@ -80,120 +80,9 @@
       </ul>
       <!-- Tab panes -->
       <div class="tab-content outline">
-        <div role="tabpanel" class="tab-pane active bulkrax-nav-tab-table-left-align" id="work-entries">
```
Comment (alishaevn) [inline app/views/bulkrax/importers/show.html.erb:83]: moved the entry tabs into partials.

Comment (ShanaLMoore) [reply app/views/bulkrax/importers/show.html.erb:83]: thanks for doing so, although both worked this looks so much better and less overwhelming to read :)

**Review by ShanaLMoore — APPROVED** [review]

**Pull request merged**